lib/Twig/Extension/Core.php

```diff
@@ -160,6 +160,7 @@ public function getFilters()
             new Twig_Filter('striptags', 'strip_tags'),
             new Twig_Filter('trim', 'trim'),
             new Twig_Filter('nl2br', 'nl2br', array('pre_escape' => 'html', 'is_safe' => array('html'))),
+            new Twig_Filter('repeat', 'str_repeat', array('pre_escape' => 'html', 'is_safe' => array('html'))),
```

Comment:
-1 for pre-escaping and being safe. You should remove both, letting the escaping being done after it.